api-reference/10 UI Widgets/dxGantt/1 Configuration/resourceAssignments/resourceAssignments.md

```diff
@@ -17,7 +17,7 @@ Resource assignments define relationship between tasks and resources.
 Use the [dataSource](/api-reference/10%20UI%20Widgets/dxGantt/1%20Configuration/resourceAssignments/dataSource.md '/Documentation/ApiReference/UI_Widgets/dxGantt/Configuration/resourceAssignments/#dataSource') option to bind the widget to a data source, which contains resource assignments. If the field names in your data source differ from the 'id', 'resourceId' and 'taskId' default names, use the [keyExpr](/api-reference/10%20UI%20Widgets/dxGantt/1%20Configuration/resourceAssignments/keyExpr.md '/Documentation/ApiReference/UI_Widgets/dxGantt/Configuration/resourceAssignments/#keyExpr'), [resourceIdExpr](/api-reference/10%20UI%20Widgets/dxGantt/1%20Configuration/resourceAssignments/resourceIdExpr.md '/Documentation/ApiReference/UI_Widgets/dxGantt/Configuration/resourceAssignments/#resourceIdExpr') and/or [taskIdExpr](/api-reference/10%20UI%20Widgets/dxGantt/1%20Configuration/resourceAssignments/taskIdExpr.md '/Documentation/ApiReference/UI_Widgets/dxGantt/Configuration/resourceAssignments/#taskIdExpr') options to map data fields.
 
 #####See Also#####
-- [Gantt Elements - Resources](concept/05%20Widgets/Gantt/10%20Gantt%20Elements/30%20Resource.md 'Documentation/Guide/Widgets/Gantt/Gantt_Elements/#Resource')
+- [Gantt Elements - Resources](/concept/05%20Widgets/Gantt/10%20Gantt%20Elements/30%20Resource.md '/Documentation/Guide/Widgets/Gantt/Gantt_Elements/#Resource')
```

Comment:
The folder with guides is called "concept**s**". It looks like you added the link to the .md file manually. Don't. Run `npm install` in the docs directory and add the usual URLs. .md links will be added automatically before you make a commit in the SourceTree.

Reply:
Yes. We've already discussed this issue with Victor. I didn't run anything manually.

Reply:
I'm afraid I cannot approve this PR because the link still looks broken to me.

Reply:
Updated links with the converter.